packages/manager/modules/exchange/src/domain/add/domain-add.controller.js

```diff
@@ -31,6 +31,7 @@ export default class ExchangeAddDomainController {
       $scope,
       wucExchange,
       ExchangeDomains,
+      coreConfig,
```

Comment:
**nitpick:** Looks like useless. Isn't it?

Reply:
`this.services.coreConfig` is used in `getDefaultLanguage`& `isFrenchLanguage`. 
I don't like to put services in `services` object but I follow the existing pattern in this file.